app/src/on-chain.ts

```diff
@@ -118,7 +118,7 @@ export class OnChain {
 
         return { deploy: deploy_accepted, executionResults };
       } catch (e) {
-        console.log("Error requesting deploy by hash from sidecar.");
+        console.log("Error requesting deploy by hash from sidecar.", e);
```

Comment:
I thought we had a winston logger built for this.  Any idea where that went?

Reply:
What's a winston logger?

Reply:
A logger I built using Winston.  It's an npm package https://www.npmjs.com/package/winston

Looks like I only added it to factoring: https://github.com/CasperLabs/factoring-dapp-backend/blob/dev/app/src/utils/logger.ts

Reply:
Cool, I'm sure we could integrate it into the middleware. Do you want to create a ticket for this?

Reply:
Yeah let's do it!